src/widget/wtrackmenu.cpp

```diff
@@ -211,6 +211,12 @@ void WTrackMenu::createActions() {
 
     if (featureIsEnabled(Feature::Properties)) {
         m_pPropertiesAct = new QAction(tr("Properties"), this);
+        // This is just for having the shortcut displayed next to the action
+        // when the menu is invoked from the tracks table.
+        // The actual shortcut is set in WTrackTableView::keyPressEvent
+        if (m_pTrackModel) {
+            m_pPropertiesAct->setShortcut(QKeySequence(tr("Ctrl+Return")));
```

Comment:
I think QKeySequence has translation under the hood, no need for() tr here.
Is there a common header where we can move it to? This allows to define it only in a single place.

Reply:
Indeed, `QKeySequence(Qt::CTRL + Qt::Key_Return)` works, too.
I'd appreciate to have one place for those shortcuts, sure. `defs_urls.h` could work, but then we'd need to complicate `WTrackTableView::keyPressEvent` to look it up, too.
Though I wonder if it's worth the hassle considering it's about one shortcut only (maybe also `Del` from #4330).

Reply:
I've put it util/defs.h, okay?

Reply:
Yes.